src/pages/[platform]/build-a-backend/storage/authorization/index.mdx

```diff
@@ -29,6 +29,12 @@ export function getStaticProps(context) {
   };
 }
 
+import flutter0 from '/src/fragments/lib/storage/flutter/gen2-banner.mdx';
+
+<Fragments
+  fragments={{ flutter: flutter0 }}
```

Comment:
We are not using Fragments anymore and have switched to `<InlineFilter>`. can you please update?